library/src/main/java/com/rengwuxian/materialedittext/TypefaceCache.java

```diff
@@ -0,0 +1,20 @@
+package com.rengwuxian.materialedittext;
+
+import android.content.res.AssetManager;
+import android.graphics.Typeface;
+
+import java.util.Hashtable;
+
+public class TypefaceCache {
+  private static final Hashtable<String, Typeface> FONT_CACHE = new Hashtable<>();
+
+  public static Typeface get(String name, AssetManager assets) {
+    synchronized (FONT_CACHE) {
```

Comment:
Hashtable is already synchronised. Does it need to be sync? the get typeface is executed on the UI thread.. and anyway I would use a weakreference to the typeface..